### ISSUE TITLE
路由设置成[绕过中国大陆地址]或[绕过局域网及中国大陆地址]，ins的图片无法显示

### ISSUE THREAD

**Issue by catscarlet** [shadowsocks/shadowsocks-android#693]
从昨天晚上开始，ins的图片就显示不出来，
尝试改了下配置，发现路由设置成[绕过中国大陆地址]或[绕过局域网及中国大陆地址]，ins的图片就无法显示，ins提示[Couldn't load image. Tap to retry]。其他应用多数都是好的，twitter和github都能正常访问。ins也只有图片和视频不能正常显示，能显示新的信息和评论。

![screenshot_2016-05-26-14-18-53](https://cloud.githubusercontent.com/assets/10594035/15565397/fd35e2a6-234c-11e6-8aef-d4154d6edb11.png)

是不是ip变了？
安卓版没有像openwrt那种可以手动配置的ignore.list，那么如何更新这个ip list？

这个问题只有北京联通宽带有，切换到北京联通3G网络会恢复。
我的两台安卓设备都遇到这个问题了。

**Comment by madeye** [comment]
Make sure you're using VPN mode, not NAT mode.

For NAT mode, you may need to flush DNS cache manually.

**Comment by catscarlet** [comment]
I'm using NAT mode.
I tried flushing DNS, but it didn't work.
I also tried [Reset], and it didn't work either.

**Comment by madeye** [comment]
Then try switch off/on airplane mode. It's an issue of Android's DNS cache mechanism. To avoid this issue, you'd better use VPN mode instead.

**Comment by catscarlet** [comment]
I just tried switching off/on airplane mode. It didn't work.
I even tried rebooting my phone and pad. It didn't work.

I don't know what had happened since yesterday night.
I reconnect my router servial times to change my outer-pi and go to company to change my network environment. The problem is still there.

I tried visiting the [image_url](https://scontent.cdninstagram.com/t51.2885-15/e35/13260805_1006035252848065_1370675413_n.jpg?ig_cache_key=MTI1Nzk4OTM0NDc1MDYxMzU0Mw%3D%3D.2), and found my phone didn't send package to my server.
